rag/svr/task_executor_refactor/task_handler.py

```
@@ -257,13 +257,31 @@ async def handle(self) -> None:
+                        CompilationTemplateService,
+                    )
+
+                    pc = self._task_context.parser_config or {}
+                    for tid in pc.get("compilation_template_ids", []):
+                        tpl = CompilationTemplateService.get_saved(tid, self._task_context.tenant_id)
+                        cfg = (tpl.get("config") or {}) if tpl else {}
+                        if isinstance(cfg, dict) and cfg.get("plan") in (True, "yes", "true"):
+                            plan_enabled = True
+                            break
+                except Exception:
+                    pass  # default to no-plan
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)